docs/Home.md

```diff
@@ -1,18 +1,71 @@
-# Welcome to Inductiva API
-
-Inductiva is a Python package for executing large-scale simulations of physical 
-systems directly in the cloud.
-
-Inductiva API offers distinct advantages:
-
-- 🔄 It consolidates various simulation domains, including fluid and molecular 
-dynamics, plasmas, and structural mechanics, under a single unified entry point.
-- 📦 Eliminates the need for installing and managing complex simulation software 
-and corresponding dependencies.
-- 🚀 Allows running hundreds or even thousands of simulations concurrently, with 
-no coding.
-- 💽 Automatically optimizes hardware configurations for each type of simulation 
-(e.g., CPU vs. GPU, appropriate number of CPU cores, RAM, etc.).
-- 🐍 With Inductiva API, you are not limited to a pre-defined GUI or intricate 
-configuration languages and scripts. Instead, you write small Python programs 
-that seamlessly integrate with your existing codebase and ML framework.
+![Inductiva_Logo_noBG(259 x 76 px)](https://github.com/inductiva/inductiva/assets/102880975/c67e31eb-2775-4ee5-ac64-027f4591c0cc)
+# Inductiva: a Python package for running large-scale simulations of physical phenomena on the Cloud.
```

Comment:
I would go even tighter:

 Inductiva: a Python package for scaling simulations on the Cloud (and on premise?)